spec/services/data_api/tad_subjects_export_spec.rb

```diff
@@ -0,0 +1,174 @@
+require 'rails_helper'
+
+RSpec.describe DataAPI::TADSubjectsExport do
+  it_behaves_like 'a data export'
+
+  describe '#data_for_export' do
+    it 'returns counts for a single application' do
+      create_application(
+        nationality: 'French',
+        domicile: 'GB',
+        application_choice_attrs: [
+          { status: :rejected, subjects: ['Mathematics'] },
+        ],
+      )
+
+      result = described_class.new.data_for_export
+
+      expect(result).to match_array([
+        {
+          subject: 'Mathematics',
+          candidate_domicile: 'UK',
+          candidate_nationality: 'EU',
+          adjusted_applications: 1,
+          adjusted_offers: 0,
+          pending_conditions: 0,
+          recruited: 0,
+        },
+      ])
+    end
+
+    it 'returns statistics for an application with split subjects' do
+      create_application(
+        nationality: 'French',
+        domicile: 'GB',
+        application_choice_attrs: [
+          { status: :rejected, subjects: ['Mathematics'] },
+          { status: :offer, subjects: ['Physics'] },
+          { status: :offer, subjects: ['Mathematics'] },
+        ],
+      )
+
+      result = described_class.new.data_for_export
+
+      expect(result).to match_array([
+        {
+          subject: 'Mathematics',
+          candidate_domicile: 'UK',
+          candidate_nationality: 'EU',
+          adjusted_applications: 0,
+          adjusted_offers: 1,
+          pending_conditions: 0,
+          recruited: 0,
+        },
+        {
+          subject: 'Physics',
+          candidate_domicile: 'UK',
+          candidate_nationality: 'EU',
+          adjusted_applications: 0,
+          adjusted_offers: 1,
+          pending_conditions: 0,
+          recruited: 0,
+        },
+      ])
+    end
+
+    it 'returns statistics for multiple applications' do
```

Comment:
Is it worth also just adding a test to check that it only returns apply1 or the latest apply again application too?

Reply:
Yes, will do...